nextjsUIapp/src/app/components/fetchYaml.tsx

```diff
@@ -0,0 +1,26 @@
+import yaml from 'js-yaml';
+import fs from 'fs';
+
+const fetchYamlData = async (filePath, slug = null) => {
```

Comment:
Where is this function used?

Reply:
seems, it's not used. So removed the code.